src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {

```

Comment:
The class name `FruitTransaction` is good, as it clearly represents the model for a transaction in the context of the fruit shop application. Make sure that the class is placed in the correct package to follow the project structure guidelines.